README.md

```diff
@@ -23,6 +23,31 @@ Try WSO2 API Gateway with direct deployment on Heroku:
 
 [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/wso2/cloud-heroku-api-gateway/tree/master)
 
+## Heroku buildpack: WSO2 API Gateway
+
+To use WSO2 API Gateway buildpack with heroku:
+
+1. Set up your app to [deploy to heroku with git](https://devcenter.heroku.com/articles/git).
+2. Set this repository as the buildpack URL:
+   
+        heroku buildpacks:set https://github.com/wso2/cloud-heroku-api-gateway.git
+3. Add the WSO2 API Cloud addon: 
+        
+        heroku addons:create wso2apicloud:<plan>
+4. Set WSO2_CLOUD_EMAIL, WSO2_CLOUD_PASSWORD, WSO2_CLOUD_ORG_KEY environment variables. This is required for setting 
+up the API Gateway.
+
+        heroku config:set WSO2_CLOUD_EMAIL=<Your WSO2 Cloud account email>
+        heroku config:set WSO2_CLOUD_PASSWORD=<Your WSO2 Cloud account password>
+        heroku config:set WSO2_CLOUD_ORG_KEY=<Your WSO2 Cloud organization key>
+
+5. Once that's done, you can deploy your app using this build pack any time by pushing to heroku:
+
+        git push heroku master
+        
+6. Finally, you need to scale your application with enough memory to run the API Gateway.
+        
+        heroku ps:scale web=1:standard-2x
 ## Documentation
 
 For more information about WSO2 API Gateway, see these WSO2 documentations:
```

Comment:
Shall we point to the correct doc here.?

Reply:
Fixed with fa437c18c4260539479f906c919dc42ea4df2580